src/ol/geom/Polygon.js

```diff
@@ -366,32 +367,35 @@ Polygon.prototype.setFlatCoordinates = function(layout, flatCoordinates, ends) {
   this.changed();
 };
 
+export default Polygon;
+
 
 /**
  * Create an approximation of a circle on the surface of a sphere.
- * @param {ol.Sphere} sphere The sphere.
  * @param {ol.Coordinate} center Center (`[lon, lat]` in degrees).
  * @param {number} radius The great-circle distance from the center to
  *     the polygon vertices.
  * @param {number=} opt_n Optional number of vertices for the resulting
  *     polygon. Default is `32`.
+ * @param {number=} opt_sphereRadius Optional radius for the sphere (defaults to
+ *     the Earth's mean radius using the WGS84 ellipsoid).
  * @return {ol.geom.Polygon} The "circular" polygon.
  * @api
  */
-Polygon.circular = function(sphere, center, radius, opt_n) {
+export function circular(center, radius, opt_n, opt_sphereRadius) {
```

Comment:
I think we need upgrade notes here as well because of the changed arguments. Hopefully not to much work, and probably helpful.

Reply:
Good point, I'll add a commit that updates the upgrade doc.